packages/config/src/options.ts

```diff
@@ -500,6 +505,12 @@ const runtimeOptions: Array<RuntimeConfigOption> = [
     validation: validate.isString,
     isInternal: true,
     canUpdateDuringTestTime: false,
+  }, {
+    name: 'additionalIgnorePattern',
```

Comment:
Looks like a new config value I had missed. Will add to the changelog. Are others able to add to this or is this internal to Cypress? I'd expect the excludeSpecPattern to be leverage instead of needing to add another option as well.

Can we scope this to the component config? it appears to be specific to component testing.

Reply:
Looking below, do we need this at all? Seems like we can just pull the specPattern and it'll resolve to either the default e2e specPttern or the custom specPattern provided: https://github.com/cypress-io/cypress/pull/20677/files#diff-0349e7b425446de9ccaf2edbb623f43cffd420688dd312850417888318b9f609R162

Reply:
This value is only used internally when we are getting the specs for the CT we want to ignore as well the specPattern that is set on the e2e property, the reason why we can just pull the specPattern is bc know that we are removing the nested properties, we are not gonna be able to know what was the value set on the e2e, the one that is going to be on the top level is the one of the CT - so we may end up ignoring the specPattern that we are looking for